framework/components/ecl-search-forms/ecl-search-forms.twig

```diff
@@ -24,52 +33,20 @@
 {% endif %}
 
 {# Print result #}
-{% if variant is defined and variant == "internal" %}
-  <form class="{{ _css_class }}" {{ _extra_attributes }}>
-    <label class="ecl-search-form__textfield-wrapper">
-      <span class="ecl-sr-only">Search this website</span>
-      {% include '@ec-europa/ecl-forms-text-inputs' with {
-        'id': 'search-form-internal',
-        'type': 'search',
-        'extraClass': 'ecl-search-form__textfield',
-        'extraAttributes': [
-          { 'name': 'title', 'value': 'Search'}
-        ]
-      } %}
-    </label>
-    {% include '@ec-europa/ecl-buttons' with {
-      'modifier': 'form-primary',
-      'label': 'Search',
-      'extraClass': 'ecl-search-form__button',
-      'extraAttributes': [
-        { 'name': 'value', 'value': 'Search'},
-        { 'name': 'type', 'value': 'submit'}
-      ]
+<form class="{{ _css_class }}"{{ _extra_attributes }}>
+  <label class="ecl-search-form__textfield-wrapper">
+    <span class="ecl-sr-only">{{ _aria_label }}</span>
+    {% include '@ec-europa/ecl-forms-text-inputs' with {
+      'id': _input.id,
+      'type': 'search',
+      'extraClass': 'ecl-search-form__textfield',
```

Comment:
`extraClass` and `extraAttributes` can be made like the others, if it's not too problematic 0 :)